.pkgr.yml

```diff
@@ -7,10 +7,15 @@ targets:
       - git
   debian-8:
     <<: *debian
+  ubuntu-12.04:
+    <<: *debian
   ubuntu-14.04:
     <<: *debian
-  ubuntu-12.04:
+  ubuntu-16.04:
     <<: *debian
+    build_dependencies:
```

Comment:
This is an example of a build without these dependencies: https://packager.io/gh/gianperrone/gogs/build_runs/1#71723

As you can clearly see, not all builds need it. The idea to add these came from looking at the log of the failing build (see link) and from looking at this history: https://github.com/pkgr/example-go-app/commits/master
Apparently bzr and mercurial are needed in all builds, but apart from Ubuntu 16.04 they are installed by default on the build images.